src/main/java/pl/touk/sputnik/Test.java

```diff
@@ -5,7 +5,10 @@
     public boolean test = false;
 
     public int method() {
+        String str = null;
         if (test) {
+            if(test = true) 
+              return str.length();
```

Comment:
[FindBugs] WARNING: NP: Null pointer dereference of str in pl.touk.sputnik.Test.method()